2-mandatory/5-reading-list.js

```diff
@@ -16,11 +16,41 @@ In this style of testing it is typical to write out as strings exactly what you
 without using any variables or any logic like loops, template strings or if statements.
 */
 
-const books = [];
+const books = [
+  {title: 'The Remains of the Day',
+  author: 'Kazuo Ishiguro',
+  alreadyRead: true,
+  },
+  {title: 'Norwegian Wood',
+  author: 'Haruki Murakami',
+  alreadyRead: true,
+  },
+  {title: 'I am a cat',
+  author: 'Natsume Soseki',
+  alreadyRead: true,
+  },
+  {title: 'Snow Country',
+  author: 'Yasunari Kawabata',
+  alreadyRead: false,
+  },
+  {title: 'The Wind-Up Bird Chronicle',
+  author: 'Haruki Murakami',
+  alreadyRead: false,
+  }
+];
   
 // exercise 1
 function logBooks() {
+  for (object of books){
+    if(object.alreadyRead ===true){
```

Comment:
Predicates can be used without the comparison for the if and else. The predicate in this case would be object.alreadyRead or !object.alreadyRead